lib/internal/util/comparisons.js

```diff
@@ -4,6 +4,7 @@ const { compare } = process.binding('buffer');
 const { isArrayBufferView } = require('internal/util/types');
 const { internalBinding } = require('internal/bootstrap/loaders');
 const { isDate, isMap, isRegExp, isSet } = internalBinding('types');
+const { getOwnNonIndicesProperties } = process.binding('util');
```

Comment:
I think this nit got lost because I had bad wifi :/ Sorry, should have been part of my previous review, but is totally non-blocking: I think this should be called `getOwnNonIndexProperties`, since the expression for the returned things in English would be “own non-index properties”

Reply:
I addressed it while landing.